cernopendata/config.py

```diff
@@ -133,6 +137,66 @@
 # Celery
 CELERY_ACCEPT_CONTENT = ["json", "msgpack", "yaml"]
 
+STATS_EVENTS = EVENTS_CONFIG
+
+STATS_AGGREGATIONS = AGGREGATIONS_CONFIG
+
+STATS_QUERIES = {
+    "record-view": {
+        "cls": TermsQuery,
+        "permission_factory": None,
+        "params": {
+            "index": "stats-record-view",
+            "doc_type": "record-view-day-aggregation",
+            "copy_fields": {
+                "recid": "recid",
+                "parent_recid": "parent_recid",
```

Comment:
I don't think you have a `parent_recid` no?

Reply:
Thanks, removed